AK/BitStream.h

```diff
@@ -27,7 +27,11 @@ class InputBitStream final : public InputStream {
         size_t nread = 0;
         if (bytes.size() >= 1) {
             if (m_next_byte.has_value()) {
-                bytes[0] = m_next_byte.value();
+                // FIXME: We don't know why this error occurs but it doesn't cause problems so we ignore it for now.
```

Comment:
This commit is redundant on master, as we've disabled `-Wmaybe-uninitialized` altogether :P

Reply:
OK!